newrelic/data_source_newrelic_notifications_destination_test.go

```diff
@@ -52,6 +73,25 @@ data "newrelic_notification_destination" "foo" {
 `, name)
 }
 
+func testAccNewRelicNotificationsDestinationDataSourceConfigByName(name string) string {
+	return fmt.Sprintf(`
+resource "newrelic_notification_destination" "foo" {
+	name = "%s"
+	type = "WEBHOOK"
+	active = true
+
+	property {
+		key = "url"
+		value = "https://webhook.site/"
+	}
+}
+
+data "newrelic_notification_destination" "foo" {
+	name = newrelic_notification_destination.foo.name
+}
```

Comment:
Similar to [this comment](https://github.com/newrelic/terraform-provider-newrelic/pull/2395/files#r1230053944) - if possible, can you please format this via `terraform fmt`

Reply:
also here - file wasn't affected by this command

Reply:
done: https://github.com/newrelic/terraform-provider-newrelic/pull/2395#discussion_r1230581080